### ISSUE TITLE
Default configuration not working

### ISSUE THREAD

**Issue by PhilETaylor** [mpclarkson/resque-bundle#8]
```
# app/config/config.yml
resque:
    class: Mpclarkson\ResqueBundle\Resque    # the resque class if different from default
    vendor_dir: %kernel.root_dir%/../vendor  # the vendor dir if different from default
    app_include: /pathto/bootstrap.php.cache # app include file if different from default

```

In your default configuration above you state to set the app_include if different from default - so I did not set it as mine is the the default location /var/bootstrap.php.cache (as per http://symfony.com/doc/master/book/performance.html)

When starting workers I get:

`APP_INCLUDE (/Users/phil/Sites/*****/app/bootstrap.php.cache) does not exist.`

So your code is trying to load the bootstrap.php.cache from the app folder and not the var folder by default, so now I'll have to set app_include to load bootstrap.php.cache from the symfony 3 default location in the var folder... when I should not have to according to your docs.

**Comment by mpclarkson** [comment]
Thanks @PhilETaylor, this has only just been updated in the last few days. @pascaldevink - any chance you can look into this?

**Comment by mpclarkson** [comment]
Fixed in commit eed3754df658d1dbad11462b40ccf1348c93a9e3. The default location for `app_include` is now `%kernel.root_dir%/../var/bootstrap.php.cache` which is the default Symfony3 location. Thanks @PhilETaylor.  Tests needed ;)
